options/rtld/generic/linker.cpp

```diff
@@ -795,6 +817,167 @@ void ObjectRepository::_parseDynamic(SharedObject *object) {
 	}
 }
 
+void ObjectRepository::_parseVerdef(SharedObject *object) {
+	if(!object->versionDefinitionTableOffset) {
+		if(verbose)
+			mlibc::infoLogger()
+				<< "mlibc: Object " << object->name
+				<< " defines no versions" << frg::endlog;
+		return;
```

Comment:
What happens in this case? Is an object still able to be resolved even if it doesn't have a DT_VERDEF?

Reply:
If an object defines no versions but was supposed to, the dependent will error out in `_parseVerneed` on line 965.

Reply:
If it defines no version and no one wants any versions from it, nothing special happens.